saleor/product/utils.py

```diff
@@ -55,10 +55,19 @@ def products_with_availability(products, discounts, local_currency):
         yield product, get_availability(product, discounts, local_currency)
 
 
-ProductAvailability = namedtuple(
-    'ProductAvailability', (
-        'available', 'price_range', 'price_range_undiscounted', 'discount',
-        'price_range_local_currency', 'discount_local_currency'))
+class ProductAvailability(namedtuple(
+    'ProductAvailability',
+        'available price_range price_range_undiscounted discount price_range_local_currency discount_local_currency')):
```

Comment:
I thought this syntax is wrong. Still, having all fields separated would be more readable and would allow better code formatting.